admin_app_2/src/app/content/edit/page.tsx

```diff
@@ -29,12 +31,13 @@ const AddEditContentPage = () => {
   const [content, setContent] = React.useState<Content | null>(null);
   const [isLoading, setIsLoading] = React.useState<boolean>(true);
 
+  const { token } = useAuth();
```

Comment:
This entire page shouldn't run any code at all (including the .getContent fetch, since it's never shown) if we're readonly and instead display a "Not Authorised" message on the page.

How do we also stop the useEffects from running on page load based on auth state?